modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java

```diff
@@ -126,7 +126,7 @@ public void processOpts() {
                         (sourceFolder + File.separator + configPackage).replace(".", java.io.File.separator), "HomeController.java"));
                 supportingFiles.add(new SupportingFile("swagger2SpringBoot.mustache",
                         (sourceFolder + File.separator + basePackage).replace(".", java.io.File.separator), "Swagger2SpringBoot.java"));
-                supportingFiles.add(new SupportingFile("application.properties",
+                supportingFiles.add(new SupportingFile("application.mustache",

```

Comment:
For Spring mvc property 'server.contextPath' means nothing, it is spring-boot related as noted here http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
So is there a reason to use same application.properties file for Spring boot and Spring mvc?